lib/vagrant-service-manager/command.rb

```diff
@@ -21,7 +21,7 @@ def exit_if_machine_not_running
   vagrant up
               eos
             @env.ui.error(message)
-            exit 1
+            exit 2
```

Comment:
The comment in this function may need to be updated.

I believe this is not an exit 2.  It either needs to be 3 (unreserved) or 1, imho.  Running this command without the machine running feels like an impermissible operation error.